tests/library/browsercontext-fetch.spec.ts

```diff
@@ -122,22 +122,40 @@ it('should add session cookies to request', async ({ context, server }) => {
 });
 
 for (const method of ['fetch', 'delete', 'get', 'head', 'patch', 'post', 'put'] as const) {
-  it(`${method} should support queryParams`, async ({ context, server }) => {
-    const url = new URL(server.EMPTY_PAGE);
-    url.searchParams.set('p1', 'v1');
-    url.searchParams.set('парам2', 'знач2');
-    const [request] = await Promise.all([
-      server.waitForRequest(url.pathname + url.search),
-      context.request[method](server.EMPTY_PAGE + '?p1=foo', {
-        params: {
-          'p1': 'v1',
-          'парам2': 'знач2',
```

Comment:
This intentionally used non-English characters to cover the encoding case, let's keep it.